src/Controller/HomeWorkController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
+
+class HomeWorkController extends AbstractController
+{
+    /**
+     * @Route("/homework/{page<\d+>}", name="home_work")
+     *
+     * @param $page
+     * @return Response
+     */
+    public function index($page = 1): Response
```

Comment:
Use less php docs in method because u will have a mess with annotations, use type hinting and return type declaration instead.
    /**
     * @Route("/homework/{page<\d+>}", name="home_work")
     */
    public function index(int $page = 1): Response

Reply:
fix `https://github.com/zrayev/sf43/pull/4/files`